Makefile

```diff
@@ -72,6 +76,9 @@ check-version:
 .PHONY: check
 check: check-version flake8 isort-check mypy
 
+.PHONY: dcheck
+dcheck: check-version flake8 isort-check dmypy
```

Comment:
This will make `check-version`, `flake8` and `isort-check` run *once* and `dmypy` run continuously. I think that might be a bit unexpected and less useful, ideally `dcheck` would continuously run `flake8` and `isort-check` too (I'd drop `check-version` since it's only really useful on the CI).

Although running those on any change might not be so easy to do, it could need some tool that we don't have installed. So I'd be OK to not add this command for now.

What do you think?

Reply:
Oh, I got if after our call.